packages/back-end/src/models/ExperimentModel.ts

```diff
@@ -802,4 +805,37 @@ export const logExperimentDeleted = async (
   return emittedEvent.id;
 };
 
-// endregion Events
+export const getAllVisualExperiments = async (
+  organization: string
+): Promise<
+  Array<{
+    experiment: ExperimentInterface;
+    visualChangeset: VisualChangesetInterface;
+  }>
+> => {
+  const visualChangesets = await findVisualChangesets(organization);
+  const allVisualChanges = flatten(
+    visualChangesets.map((vc) => vc.visualChanges)
+  );
+  const experiments = (
+    await ExperimentModel.find({
+      id: {
+        $in: visualChangesets.map((changeset) => changeset.experiment),
+      },
+      organization,
+      archived: false,
+    })
+  ).filter(
+    // If the experiment is stopped and the released variation doesn’t have any visual changes, it should be excluded
+    (e) =>
+      e.status !== "stopped" ||
+      allVisualChanges.some((vc) => vc.variation === e.releasedVariationId)
```

Comment:
The variation ids are not globally unique, just unique within a single experiment.  So this may have false positives.

Also, I'm not sure this will work as expected.  If the experiment loses, `releasedVariationId` will be the control, which will technically have visual changes, but they will be empty (`css` is empty string, `dom` is an empty array).  We want to filter that out as well.

Reply:
Refactored to group by experiment, also filtering out losing experiments.